subsys/net/lib/openthread/platform/spi_interface.cpp

```diff
@@ -0,0 +1,342 @@
+/*
+ *  Copyright (c) 2019, The OpenThread Authors.
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
```

Comment:
This is a problem, AFAIK no license other than Apache-2.0 is allowed in Zephyr tree.

Reply:
Yeah, I figured that much. If this blocks the PR, I could move this into the [zephyrproject/openthread repo](https://github.com/zephyrproject-rtos/openthread).
But then the code is split across repositories.

Reply:
That's also an option, although still not perfect, as we're aiming to keep the fork as close to the original repo as possible. Pehrpas you could ask the OT community if they're if we reuse parts of the code under Zephyr license? It seems that there's not that much overlap between the original file and the version you've posted here.